app/js/main.js

```diff
@@ -17,9 +17,9 @@
 import {DemoPaywallController} from './demo-controller';
 import {log} from './log';
 
-
 log('started');
 
+const AVAILABLE_SKUS = ['basic_trial', 'basic_1', 'basic_monthly', 'annual_1'];
```

Comment:
Since this list is only used in one flow, could you move this variable definition to that flow's `if` statement?